src/Rules.UnitTests/EmbeddedResourceTests.cs

```diff
@@ -58,6 +61,44 @@ public void CheckEmbeddedDescriptionFiles_ByLanguage(string languageKey, int atL
             resourceNames.Should().HaveCountGreaterThan(atLeastXRules);
         }
 
+        [TestMethod]
+        public void CheckEmbeddedDescriptionFiles_AreParseableAsXml()
```

Comment:
Tests loading all of the embedded rule descriptions and making sure we can parse them as XML.

This is a safety net in case the format of the generated rule description HTML changes - this test will fail if we update to a newer version of an analyzer with a new-format rule description.